* Respect CFLAGS, don't strip binaries
* Don't check for configure.in (EAPI 8)

http://bugs.gentoo.org/252124

--- a/configure.in
+++ b/configure.in
@@ -1,6 +1,6 @@
 dnl Process this file with autoconf to produce a configure script.
 
-AC_INIT(configure.in)
+AC_INIT
 AM_INIT_AUTOMAKE(selectwm, 0.4.1)
 AM_CONFIG_HEADER(config.h)
 AM_MAINTAINER_MODE
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -10,5 +10,6 @@
 	modify.c modify.h \
 	misc.h define.h
 
-LIBS = @GTK_LIBS@
-AM_CFLAGS = -s -O2 -Wall @GTK_CFLAGS@ -DLOCALEDIR=\"$(datadir)/locale\"
+selectwm_LDADD = @GTK_LIBS@
+selectwm_CFLAGS = @GTK_CFLAGS@
+selectwm_CPPFLAGS = -DLOCALEDIR=\"$(datadir)/locale\"
